php/swoole_fast-route/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM twosee/swoole-coroutine:latest
```

Comment:
I think `zaherg/php-swoole`  is more accurate

Reply:
at least it use `pecl` -> allowing us to not use `swoole`  **master** but the last stable (4.3 as for now)

Reply:
It's a bad idea. 
In my test this leads to performance reduce. (It's maybe alpine vs debian or pecl vs compile or etc problems).
On my hw:
1st:
```
Requests / s | Throughput |
138286.67 | 350.80 MB |
```
2nd:
```
Requests / s | Throughput |
81526.33 | 206.29 MB |
```

Reply:
sure, I understand the performance gap, but using **master** leads to results fluctuating, I mean we can show any version